### PR TITLE
Fix updating homebrew languages

### PR DESCRIPTION
**Pull request by CarlosFdez**
The render in the languages update method was eating the update to the languages itself as well, and since updateObject always leads to a re-render it was unnecessary.

### CODE REVIEW

File: src/module/system/settings/homebrew/languages.ts [hunk 1]
```diff
@@ -107,7 +107,10 @@ export class LanguagesManager {
         if (oldRarity === newRarity) return;
 
         const language = droppedEl.dataset.slug;
-        if (!objectHasKey(CONFIG.PF2E.languages, language) || language === "common") {
+        const isValidLanguage =
+            language &&
+            (objectHasKey(CONFIG.PF2E.languages, language) || this.menu.cache.languages.some((l) => l.id === language));
```
Comment (stwlam) [inline src/module/system/settings/homebrew/languages.ts:112]: This looks like an unrelated change

File: src/module/system/settings/homebrew/languages.ts [hunk 2]
```diff
@@ -155,22 +158,18 @@ export class LanguagesManager {
...
 
-        let render = false;
-
         if (
             source.commonLanguage &&
             !languageSet.has(source.commonLanguage) &&
             !updatedLanguages.includes(source.commonLanguage)
         ) {
             source.commonLanguage = null;
-            render = true;
```
Comment (stwlam) [inline src/module/system/settings/homebrew/languages.ts:166]: Ditto

File: static/templates/system/settings/homebrew.hbs [hunk 3]
```diff
@@ -11,7 +11,7 @@
         <div class="form-group setting">
             <label for="{{options.id}}-languages">{{localize settings.languages.name}}</label>
             <div class="form-fields">
-                <input type="text" name="languages" class="homebrew pf2e-tagify" data-dtype="JSON" value="{{json traitSettings.languages.value}}">
+                <input type="text" name="languages" class="homebrew pf2e-tagify" data-dtype="JSON" value="{{json settings.languages.value}}">
```
Comment (stwlam) [inline static/templates/system/settings/homebrew.hbs:14]: This is the fix, right?

File: src/module/system/settings/homebrew/languages.ts (hunk 1 above)
Comment (CarlosFdez) [reply src/module/system/settings/homebrew/languages.ts:112]: Only marginally unrelated. There are two issues with updating homebrew languages.

1) Adding new ones
2) setting the rarity of freshly added ones before saved

This would be one entry in the changelog.

File: static/templates/system/settings/homebrew.hbs (hunk 3 above)
Comment (CarlosFdez) [reply static/templates/system/settings/homebrew.hbs:14]: No. Its still broken even with just this change. I think something changed in core that caused the rerender of the rarity update to not reflect changes in the languages themselves.

File: src/module/system/settings/homebrew/languages.ts (hunk 1 above)
Comment (CarlosFdez) [reply src/module/system/settings/homebrew/languages.ts:112]: I can try without this, if you'd rather I split this off, though I can't really see this being a second changelog entry.
